.github/workflows/CI.yml

```diff
@@ -53,6 +53,12 @@ jobs:
       - name: Build job-scheduler
         working-directory: ./job-scheduler
         run: ./gradlew publishToMavenLocal -Dopensearch.version=1.0.0-alpha1 -Dbuild.snapshot=false
+      - name: Assemble job-scheduler
```

Comment:
From the error of https://github.com/opensearch-project/anomaly-detection/pull/25/checks?check_run_id=2440286543
> Could not find com.amazon.opendistroforelasticsearch:opendistro-job-scheduler-spi:1.0.0.0.

We should build job scheduler SPI as well.

Reply:
The real issue here is that github checkout deletes everything in the current folder when checking out 

```
Deleting the contents of '/home/runner/work/anomaly-detection/anomaly-detection'
```